core/trino-main/src/main/java/io/trino/dispatcher/FailedDispatchQuery.java

```diff
@@ -262,6 +262,8 @@ private static QueryStats immediateFailureQueryStats()
                 new Duration(0, MILLISECONDS),
```

Comment:
commit message is too long

Is this two changes in single commit? Could we split them?

Reply:
only metric changes which can not split more.

Reply:
commit message is still too long